plugins/core/sdk-codegen/codegen-resources/codewhispererruntime/service-2.json

```diff
@@ -1019,12 +1016,6 @@
                 },
                 "codeGenerationStatus": {
                     "shape": "CodeGenerationStatus"
-                },
-                "codeGenerationRemainingIterationCount": {
-                    "shape": "Integer"
-                },
-                "codeGenerationTotalIterationCount": {
-                    "shape": "Integer"
```

Comment:
this was added in https://github.com/aws/aws-toolkit-jetbrains/pull/4662

Reply:
Yea, I see now. I've got the latest from main, pushed now in the next rev.